storops_test/unity/rest_data/userQuota/user_quota_1.json

```diff
@@ -0,0 +1,17 @@
+{
+    "content": {
+        "existed": true,
+        "filesystem": {
+            "UnityFileSystem": {
```

Comment:
This response body is wired, Unity shouldn't return words like "UnityFileSystem" which defined in storops.
Is this body returned by Unity? I remember the correct body should like:
```
        "filesystem": {
              "id": "fs_36"
        },
```

Reply:
Hi @yong-huang , I think I pasted the response from Ansible module by mistake. Will update it as needed. Thanks